src/core_plugins/tagcloud/public/__tests__/tag_cloud.js

```diff
@@ -102,11 +102,16 @@ describe('tag cloud', function () {
       tagCloud.setData(test.data);
       tagCloud.setOptions(test.options);
       tagCloud.on('renderComplete', function onRender() {
```

Comment:
I think these tests would be a lot easier to follow if they were using promises, and as a bonus errors would already be propagating correctly.